.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
 	extends: [
 		'plugin:@wordpress/eslint-plugin/recommended',
 		'plugin:jest/recommended',
+		'plugin:eslint-comments/recommended',
```

Comment:
Does order of those configs matter?

Reply:
Not that I am aware of. Can't find anything in the docs that would indicate this.

Reply:
I'm not certain here but I want to say that order does matter. Because what if 2 plugins set configuration for a single rule in different ways?

Reply:
I ran a quick test `eslint --print-config .eslintrc.js`

https://gist.github.com/ntwb/d04000c8a29bd441c05b20650c027ed8/revisions

The order of the rules is changed, but, I agree that there is a chance of conflicting rules having some unknown behavior.

Reply:
It looks like we are good here with the current order. It's a bit surprising though that rules are applied this way. I assumed the opposite.